tests/test_live.py

```diff
@@ -0,0 +1,1428 @@
+#!/usr/bin/env python
+"""Test gs-wrap live."""
+
+# pylint: disable=missing-docstring
+# pylint: disable=too-many-lines
+# pylint: disable=protected-access
+# pylint: disable=expression-not-assigned
+
+import concurrent.futures
+import os
+import pathlib
+import shutil
+import subprocess
+import time
+import unittest
+import uuid
+import warnings
+from typing import List
+
+import google.api_core.exceptions
+import temppathlib
+
+import gswrap
+
+# test enviroment bucket
+TEST_GCS_BUCKET = None  # type: str
+TEST_GCS_BUCKET_NO_ACCESS = None  # type: str
+NO_WARNINGS = True  # type: bool
+VERBOSE = False  # type: bool
+GCS_FILE_CONTENT = "test file"  # type: str
+
+
+def gcs_test_setup(prefix: str):
```

Comment:
please write a short description

`# Create test folders structure to be used in the live test `